Makefile

```diff
@@ -122,33 +122,42 @@ init: restart
 # Targets for testing ibis inside docker's containers
```

Comment:
The diff checker got a little confused by these changes, so the diff here doesn't really make sense

Reply:
Quick links to the versions before and after:
[Makefile (before)](https://github.com/ibis-project/ibis/blob/5be78af0a6d1cd56a6ceb89067c43069d18084a6/Makefile#L124)
[Makefile (after)](https://github.com/ibis-project/ibis/blob/cdba6b7a924f557f937a7e3edfe6f26582e173dd/Makefile#L124)